### ISSUE TITLE
Optiboot 328p 8MHZ int bootloader

### ISSUE THREAD

**Issue by fifa255** [Optiboot/optiboot#195]
I am using the boards manager to install the optiboot bootloader library.

I am able to burn the bootloaders succesfully using the arduino as ISP as detailed in the link below
https://www.arduino.cc/en/Tutorial/ArduinoToBreadboard

I am using the optiboot 328p 8MHZ (int) bootloader.  I am able to burn this onto my standalone 328p successfully.

However if the use the optiboot bootloader I am able to upload a sketch to the 328p only once. After that it gives me the error message "avrdude: stk500_getsync() attempt 1 of 10: not in sync: resp=0x00".

I have tried to upload the UNO bootloader that comes with the Arduino IDE and I have no problems uploading multiple sketches to the chip with that bootloader.

**Comment by fifa255** [comment]
I am using Arduino IDE 1.6.9

**Comment by WestfW** [comment]
"able to upload only once" is usually an indication of wrong fuses (not BOOTRST) or non-working auto-reset circuitry.  What have you done to implement auto-reset?

**Comment by fifa255** [comment]
I am using an arduino UNO board as ISP to program the 328p. The reset pin from the UNO board is connected to the reset pin on 328p as shown at 
https://www.arduino.cc/en/Tutorial/ArduinoToBreadboard

I am able to upload the regular UNO bootloader onto the 328p without problems and then able to upload multiple sketches using the same circuitry.  If I use the optiboot bootloader it fails after the first upload.

**Comment by dragan1alex** [comment]
Can confirm, I have the same issue as well on an Atmega88P clocked at 8MHz, maybe the configuration file at "AppData\Local\Arduino15\packages\arduino\tools\avrdude\6.3.0-arduino9/etc/avrdude.conf" is not correctly written (?). The first time trying to upload a program I get some unexpected device signature, but I can burn the program into the chip by copying the command from the arduino IDE in the windows command prompt and adding the "-F" argument. After the first program upload it doesn't sync anymore for any further uploads so I need to burn the bootloader after each upload (which is annoying).

**Comment by cimba007** [comment]
I ran into a "similar" problem, although might not be related at all. I use the boardsmanager of arduino too for my atmega328p.

Now to the issue:

In my setup-method I perform a voltage check of the battery .. if it is too low then I go to a endless-sleep-loop (with 8S watchdog).

Now my problem is that for testing purpose I tried to go to sleep on every boot .. and doing so I bricked my bootloader (had to reflash bootloader again).

The sketch was running fine .. but I was no longer able to flash new sketches ..
